clients/fides-js/src/components/Toggle.tsx

```diff
@@ -28,7 +28,7 @@ const Toggle = ({
         onChange={() => {
           onChange(id);
         }}
-        defaultChecked={checked}
+        checked={checked}
```

Comment:
this is kind of a weird thing, but it turns out `defaultChecked` only provides the initial value. this started not to work when I added the "select all/deselect toggles" which wanted to control _other_ toggles' state. This seems to work well, though I had to update the cypress tests now that the `checked` prop is a lil different

more info: https://stackoverflow.com/questions/33131413/react-defaultchecked-dont-render-check-attribute-on-second-load